src/dashboard/src/utils/request.js

```diff
@@ -25,8 +26,13 @@ const codeMessage = {
  * 异常处理程序
  */
 const errorHandler = error => {
-  const { response = {} } = error;
-  const errortext = codeMessage[response.status] || response.statusText;
+  const { response, data } = error;
+  const errortext = (
+    <>
+      {codeMessage[response.status] || response.statusText} <br />
+      {data.msg[0]}
```

Comment:
Can you confirm that there must be data in the incorrect response information? There must be msg in the data, and msg must be an array. If the format cannot be determined, it is recommended to increase the judgment, otherwise the front-end may encounter exceptions.

Reply:
Does that mean we need to use `if` to check the response content? 
If there is no msg or msg is not an array, we display the default message?

I think most of the backend error handlers send data msg array.

Reply:
now add condition check:

`{data && data.msg && Array.isArray(data.msg) && data.msg.length > 0 && data.msg[0]}`